src/channel_state.ts

```diff
@@ -437,6 +437,51 @@ export class ChannelState<
     return messageWithReaction;
   }
 
+  removeQuotedMessages(
+    message: MessageResponse<
+      AttachmentType,
+      ChannelType,
+      CommandType,
+      MessageType,
+      ReactionType,
+      UserType
+    >,
+  ) {
+    const parseMessage = (
```

Comment:
@vishalnarkhede second time this function is needed (the other one was in the sdk) maybe we can put it on utils or something?